conformance/fhir-ig-davinci-hrex/README.md

```diff
@@ -1,8 +1,8 @@
-# HL7 FHIR� Da Vinci Health Record Exchange (HRex) 0.2.0 - STU R1 - 2nd ballot
+# HL7 FHIR� Da Vinci Health Record Exchange (HRex) 1.0.0 - STU R1
```

Comment:
what char is this?  would have expected ® like before

Reply:
Hmm...  Not sure what happened here -- I will fix.